src/views/CreateOrUpdateRequest.svelte

```diff
@@ -178,8 +178,8 @@ function onWeightChanged(event) {
     </div>
     
     {#if imageUrl || request.photo && request.photo.url}
-      <div class="col-12 col-sm-5 image-preview text-center text-sm-left">
-        <img src={imageUrl || request.photo && request.photo.url} alt="Your image" class="image-preview" />
+      <div class="col-12 col-sm-5 text-center text-sm-left">
+        <img src={imageUrl || request.photo && request.photo.url} alt="Request details" class="preview" />
```

Comment:
Why an alt-text of "Request details"?

Reply:
can't use the word picture, image, photo...A11y complains :-D  it thinks you're being redundant to screen readers...which we're not, it's silly.